packages/react-native-tab-view/src/TabBarIndicator.tsx

```diff
@@ -120,19 +120,35 @@ export function TabBarIndicator<T extends Route>({
     );
   }
 
+  const styleList = [];
+
+  // scaleX doesn't work on chrome and opera for linux, including their
+  // versions for android
+  if (Platform.OS === 'web' && width === 'auto') {
```

Comment:
Maybe we could detect the operating system through `window.navigator.userAgent`?  Because it does not apply to MacOS / Windows.

Reply:
this is not good for SSR